packages/backend/src/tenants/service.ts

```diff
@@ -39,13 +40,18 @@ export async function createTenantService(
 
 async function getTenant(
   deps: ServiceDependencies,
-  id: string
+  id: string,
+  includeDeleted: boolean
```

Comment:
small nit:

```suggestion
  includeDeleted: boolean = false
```
(instead of having this on L31, since its more clear when you are reading the function)